config/deploy/templates/db_migrate_hook.yaml.erb

```diff
@@ -1,40 +0,0 @@
-apiVersion: v1
-kind: Pod
-metadata:
-  name: "huddle-db-migrate-<%= deployment_id %>"
```

Comment:
Looks like this hook may need to be removed. Not sure why this didn't trigger in the PR's CI.